fast64_internal/f3d/f3d_parser.py

```diff
@@ -2244,24 +2277,41 @@ def getImportData(filepaths):
 
 
 def parseMatrices(sceneData: str, f3dContext: F3DContext, importScale: float = 1):
-    for match in re.finditer(rf"Mtx\s*([a-zA-Z0-9\_]+)\s*=\s*\{{(.*?)\}}\s*;", sceneData, flags=re.DOTALL):
-        name = "&" + match.group(1)
-        values = [hexOrDecInt(value.strip()) for value in match.group(2).split(",") if value.strip() != ""]
-        trueValues = []
-        for n in range(8):
-            valueInt = int.from_bytes(values[n].to_bytes(4, "big", signed=True), "big", signed=False)
-            valueFrac = int.from_bytes(values[n + 8].to_bytes(4, "big", signed=True), "big", signed=False)
-            int1 = values[n] >> 16
-            int2 = int.from_bytes((valueInt & (2**16 - 1)).to_bytes(2, "big", signed=False), "big", signed=True)
-            frac1 = valueFrac >> 16
-            frac2 = valueFrac & (2**16 - 1)
-            trueValues.append(int1 + (frac1 / (2**16)))
-            trueValues.append(int2 + (frac2 / (2**16)))
+    finditer = list(re.finditer(rf"Mtx\s*([a-zA-Z0-9\_]+)\s*=\s*\{{(.*?)\}}\s*;", sceneData, flags=re.DOTALL))
+
+    # newer assets system
+    if len(finditer) == 0:
+        finditer = list(re.finditer(r"Mtx\s*([a-zA-Z0-9\_]+)\s*=\s*(.*?)\s*;", sceneData, flags=re.DOTALL))
 
+    for match in finditer:
+        name = "&" + match.group(1)
+        data = match.group(2)
         matrix = mathutils.Matrix()
-        for i in range(4):
-            for j in range(4):
-                matrix[j][i] = trueValues[i * 4 + j]
+
+        if "#include" in data:
+            match = re.search(r".*\((.*?)\)", get_include_data(data, strip=True), re.DOTALL | re.MULTILINE)
+            assert match is not None
+            raw_matrix = match.group(1).split(",")
+            for i in range(4):
+                for j in range(4):
+                    matrix[j][i] = float(raw_matrix[i * 4 + j].removesuffix("f"))
```

Comment:
```suggestion
                    matrix[i][j] = float(raw_matrix[i * 4 + j].removesuffix("f"))
```
this fixes jabu afaict